tools/src/versioning/ios/transforms/postTransforms.ts

```diff
@@ -77,6 +77,13 @@ export function postTransforms(versionName: string): TransformPipeline {
         replace: new RegExp(`(React)\\/${versionName}(bridging)\\/`, 'g'),
         with: `$1/$2/${versionName}`,
       },
+      {
+        // Codegen adds methods to `RCTCxxConvert` that start with `JS_`, which refer to `JS::`
+        // C++ namespace that we prefix, so these methods must be prefixed as well.
+        paths: 'FBReactNativeSpec-generated.mm',
```

Comment:
should we also transform `FBReactNativeSpec-generated.h`?

Reply:
You're right 🤦 Thank you!
```suggestion
        paths: ['FBReactNativeSpec.h', 'FBReactNativeSpec-generated.mm'],
```